packages/tokens/properties/color/font.json

```diff
@@ -1,16 +0,0 @@
-{
-  "eds": {
```

Comment:
these are totally unused in the app, and we're about to move to a new token structure anyways